plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/BannedElementTest.java

```diff
@@ -27,7 +27,7 @@ public void testBannedElementConstructor_nullElementName() {
 
   @Test(expected = NullPointerException.class)
   public void testBannedElementConstructor_nullLocation() {
-    new BannedElement("test", null, 0);
+    new BannedElement("test", "", 1, null, 0);
```

Comment:
Marker IDs are only used when searching the resource for markers for quick fixes. Should I add my own check? IResource.createMarker(marker ID) doesn't have any checks.